src/Command/AnalyseCommand.php

```diff
@@ -122,7 +122,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		if ($generateBaselineFile === false) {
 			$generateBaselineFile = null;
 		} elseif ($generateBaselineFile === null) {
-			$generateBaselineFile = 'phpstan-baseline.neon';
+			$generateBaselineFile = file_exists('phpstan-baseline.php') ? 'phpstan-baseline.php' : 'phpstan-baseline.neon';
```

Comment:
micro optimazation, which requires less syscalls
```suggestion
			$generateBaselineFile = is_file('phpstan-baseline.php') ? 'phpstan-baseline.php' : 'phpstan-baseline.neon';
```

Reply:
thank you, didn't know, updated